esmvaltool/recipes/recipe_psyplot.yml

```diff
@@ -0,0 +1,63 @@
+# ESMValTool
+# recipe_psyplot.yml
+---
+documentation:
+  title: >
+    Create arbitrary Psyplot plots.
+
+  description: >
+    This recipes showcases the use of the Psyplot diagnostic that provides a
+    high-level interface to Psyplot for ESMValTool recipes. For each input
+    dataset, an individual plot is created.
+
+  authors:
+    - schlund_manuel
+
+  maintainer:
+    - schlund_manuel
+
+  references:
+    - sommer17joss
+
+  projects:
+    - 4c
+    - esm2025
+    - isenes3
+    - usmile
+
+
+datasets:
+  - {project: CMIP6, dataset: CanESM5,     exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 1995, end_year: 2014}
+  - {project: CMIP6, dataset: ICON-ESM-LR, exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 1995, end_year: 2014}
```

Comment:
I got an error for the ICON-ESM-LR model running the recipe as it is:
```
esmvalcore.cmor.check.CMORCheckError: There were errors in variable tas:
 Coordinate longitude has var name longitude instead of lon
 Coordinate latitude has var name latitude instead of lat
```

using other CMIP6 models and a native ICON file worked well.

Reply:
Thanks for testing @lukruh!! You need to update your ESMValCore for this, I recently merged a fix for ICON-ESM-LR (https://github.com/ESMValGroup/ESMValCore/pull/1582).

Reply:
I updated the core and its working fine! Well done definitively useful for me.